operations/overlay.yml

```diff
@@ -0,0 +1,9 @@
+id: overlay
+name: overlay
+description: This operation adds additional qedges and/or kedges.
```

Comment:
Might overlay just simply add Attributes to existing nodes or edges without adding any?

Reply:
@edeutsch iirc, the idea is _not_ to have overlay add attributes. The rational is that each edge has a `provided_by` and `defined_by`, and we don’t want to be adding additional information that wasn’t provided or defined by what is quoted. Eg. We want to add edges that have `probably_treats` or `chi_square` or `has_ngd_with` independently from the edges that exist in the KG.

Reply:
okay, that seems sensible for edges.

What about for nodes? Is there an operation that might provide more information as Attributes for Nodes? Is that an overlay operation?

Reply:
In our list of potential operations we actually identified several different overlays:

Overlay (new edges)
Overlay (edge annotations)
Overlay (node annotations)
Overlay (edge bindings)

We didn't give them different names, I wonder if we should make the  id/name of this operation more specific.

Reply:
I agree that adding attributes to existing edges/nodes should be a separate operation ("annotate"?).